VimeoNetworking/VimeoNetworking/ResponseCache.swift

```diff
@@ -35,22 +48,18 @@ final internal class ResponseCache
     {
         let key = request.cacheKey
         
-        if let responseDictionary = self.memoryCache.responseDictionaryForKey(key)
-        {
-            completion(result: .Success(result: responseDictionary))
-        }
-        else
-        {
-            self.diskCache.responseDictionaryForKey(key) { responseDictionary in
-                
-                if let responseDictionary = responseDictionary
-                {
+        self.memoryCache.responseDictionary(forKey: key) { (responseDictionary) in
```

Comment:
I don't think standardizing the protocol is worth the cost of making this a closure instead of a return val